examples/blocksparse_attention/block_sparse_attn_triton.py

```
@@ -192,24 +191,26 @@ def _fwd_kernel(
-             k,
-             v,
-             block_sparse_mask,
-             sm_scale,
-             BLOCK_M=64,
-             BLOCK_N=64,
-             num_warps=None,
-             num_stages=1,
-             out=None):
-
+def _forward(
+    ctx,
```

Ruff (0.14.0)

Unused function argument: `ctx`

(ARG001)